javascript/src/seedsource/containers/Map.jsx

```diff
@@ -605,10 +608,41 @@ class Map extends React.Component {
         }
     }
 
+    updateVectorLayers(layers) {
+        let numLayersToAdd = layers.length - this.displayedVectorLayers.length
+        if (this.displayedVectorLayers.map(layer => layer._url).toString() === layers.map(layer => layer.urlTemplate).toString()) {
+            return
+        }
+
+        let rewriteLeafletVectors = () => {
```

Comment:
This is only used once, and it's an inline function already. Go ahead and just inline the code where it's used below.